src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -472,43 +470,15 @@ def _construct_job_status_set(self, job_ids: list) -> dict:
                 kblogging.log_event(
                     self._log, "construct_job_status_set", {"err": str(e)}
                 )
+
         for job_id, state in fetched_states.items():
             revised_state = self._construct_job_status(self.get_job(job_id), state)
             if revised_state["state"]["status"] in TERMINAL_STATES:
                 self._completed_job_states[job_id] = revised_state
             job_states[job_id] = revised_state
         return job_states
 
-    def _verify_job_parentage(self, parent_job_id, child_job_id):
-        """
-        Validate job relationships.
-        1. Make sure parent exists, and the child id is in its list of sub jobs.
-        2. If child doesn't exist, create it and add it to the list.
-        If parent doesn't exist, or child isn't an actual child, raise an exception
-        """
-        if parent_job_id not in self._running_jobs:
-            raise ValueError(
-                "Parent job id {} not found, cannot validate child job {}.".format(
-                    parent_job_id, child_job_id
-                )
-            )
-        if child_job_id not in self._running_jobs:
-            parent_job = self.get_job(parent_job_id)
-            parent_state = parent_job.state()
-            if child_job_id not in parent_state.get("sub_jobs", []):
-                raise ValueError(
-                    "Child job id {} is not a child of parent job {}".format(
-                        child_job_id, parent_job_id
-                    )
-                )
-            else:
-                self._create_jobs([child_job_id])
-                # injects its app id and version
-                child_job = self.get_job(child_job_id)
-                child_job.app_id = parent_job.meta.get("batch_app")
-                child_job.tag = parent_job.meta.get("batch_tag", "release")
-
-    def lookup_job_info(self, job_id, parent_job_id=None):
```

Comment:
remove these `parent_job_id` arguments throughout as they are not useful (and not implemented in the FE)